### ISSUE TITLE
IndexOutOfBounds when running an add command without sufficient parameters

### ISSUE THREAD

**Issue by IncompetentDev** [AY2122S2-CS2113-F10-2/tp#42]
Reproduction steps:

1. Enter `add`.
2. ArrayIndexOutOfBoundsException will be thrown.
3. Similarly, enter `add n/Paper Cup`

```
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
        at seedu.duke.parser.Parser.getName(Parser.java:62)
        at seedu.duke.parser.Parser.createItem(Parser.java:99)
        at seedu.duke.parser.Parser.parse(Parser.java:32)
        at seedu.duke.InvMgr.run(InvMgr.java:45)
        at seedu.duke.InvMgr.main(InvMgr.java:58)
```
Possible cause:

```
String[] itemString = userCommand.split("/");
String name = getName(itemString);
int quantity = getQuantity(itemString);
String desc = getDesc(itemString);
```

Each get attribute call uses explicit indexing, eg name is `itemString[1]`, quantity is `itemString[2]`.
itemString does not return a sufficiently long array.

**Comment by IncompetentDev** [comment]
Maybe fix for 2.0

**Comment by IncompetentDev** [comment]
AB3 Parser does not use hard-coded indexing, so this issue is now resolved.
